plerkle_serialization/Cargo.toml

```
@@ -1,7 +1,7 @@
 [package]
 name = "plerkle_serialization"
 description = "Metaplex Flatbuffers Plerkle Serialization for Geyser plugin producer/consumer patterns."
-version = "1.8.0"
+version = "1.9.0"
```

There is no actual changes, and this version will be fully compatible with the 1.8.0, right?
```suggestion
version = "1.8.1"
```

---

@StanChe I decided to use `1.9.0` version because we updated both `solana-sdk ` and `solana-transaction-status` to `1.18.11` version, so to my it's looks like major update so it make since to change second octave in version, not the third one.